package.json

```diff
@@ -1,22 +1,26 @@
 {
-  "name": "node.js_course_practice",
-  "version": "1.0.0",
-  "description": "",
-  "main": "app.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/DimitriIlnitski/Node.js_Course_Practice.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/DimitriIlnitski/Node.js_Course_Practice/issues"
-  },
-  "homepage": "https://github.com/DimitriIlnitski/Node.js_Course_Practice#readme",
-  "dependencies": {
-    "express": "^4.18.2"
-  }
+	"name": "node.js_course_practice",
+	"version": "1.0.0",
+	"description": "",
+	"main": "app.js",
+	"scripts": {
+    "start": "node app.js",
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"generate-swagger": "node swagger.js"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/DimitriIlnitski/Node.js_Course_Practice.git"
+	},
+	"author": "",
+	"license": "ISC",
+	"bugs": {
+		"url": "https://github.com/DimitriIlnitski/Node.js_Course_Practice/issues"
+	},
+	"homepage": "https://github.com/DimitriIlnitski/Node.js_Course_Practice#readme",
+	"dependencies": {
+		"express": "^4.18.2",
+		"swagger-jsdoc": "^6.2.8",
+		"swagger-ui-express": "^5.0.0"
+	}
```

Comment:
Within the scope of your task, you need to install the 'nodemon' package

Reply:
I have installed nodemon globally, that's why it was not in package.json. How I have installed it for dev mode.